src/main/java/com/solopov/urlshorter/web/controllers/UrlShorterController.java

```diff
@@ -0,0 +1,42 @@
+package com.solopov.urlshorter.web.controllers;
+
+import com.solopov.urlshorter.exceptions.LinkNotFoundException;
+import com.solopov.urlshorter.sevices.UrlShorterService;
+import com.solopov.urlshorter.web.dto.UrlGenerateRequestDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.net.URI;
+
+import static org.apache.tomcat.websocket.Constants.MOVED_PERMANENTLY;
+
+@RestController
+@RequestMapping("${com.solopov.urlshorter.service.api.shorten.url}")
```

Comment:
any reasons it's read from properties?

Reply:
That works :)

Reply:
upd coommit: left th same. because it works